tests/unit-tests/dds/DCPS/GuidGenerator.cpp

```diff
@@ -43,30 +43,37 @@ TEST(dds_DCPS_GuidGenerator, maintest)
     gen.populate(g2);
     EXPECT_TRUE(not_null(g1));
     EXPECT_TRUE(compare_prefix(g1,g2) != 0);
-  }
+
+
+    int val1 = gen.interfaceName("Test_NotARealInterface");
```

Comment:
Please change `TEST(dds_DCPS_GuidGenerator, maintest)` to `TEST(dds_DCPS_RTPS_GuidGenerator, populate)`.

Please move the new test to `TEST(dds_DCPS_RTPS_GuidGenerator, interfaceName)`.